spec/pagseguro/features/create_transaction_request_spec.rb

```diff
@@ -37,8 +37,8 @@
         status: [400, "Bad Request"], body: body, content_type: "text/xml"
     end
 
-    it "returns false" do
-      expect(transaction.create).to be_falsey
+    it "returns transaction" do
+      expect(transaction.create).to eq(transaction)
```

Comment:
@dmitryrck I thought we could follow the activerecord way. when `save` is called it returns `true` or `false`. when `create` is called, it's already on a instance.